src/__tests__/ModalAmount.test.tsx

```diff
@@ -1,42 +1,47 @@
 import React from 'react';
-import { ModalPaymentProvider } from '../utilities/hooks/ModalPaymentContext/context';
-import ModalPage, { Props } from '../components/ModalAmount';
-import { render } from '@testing-library/react';
+import { cleanup, fireEvent } from '@testing-library/react';
+import * as ModalPaymentContext from '../utilities/hooks/ModalPaymentContext';
+// import { ModalPaymentProvider } from '../utilities/hooks/ModalPaymentContext';
```

Comment:
nit: this and some other still-commented code, should we remove before checking in?